src/main/drivers/inverter.c

```diff
@@ -75,37 +88,37 @@ void enableInverter(USART_TypeDef *USARTx, bool on)
 
 #ifdef INVERTER_PIN_UART1
```

Comment:
Functions now take `identiier` instead of `pin`.
Haven't gone far to use `serialPort_t` nor `uartPort_t`; leave it for further refinement.